src/test/java/org/itsallcode/openfasttrace/importer/legacytag/config/DescribedPathMatcherTest.java

```diff
@@ -90,21 +90,52 @@ public void testListBasedMatcherMatchesAbsoluteWindowsStylePathWithForwardSlash(
     {
         createListMatcher("C:/abs/path");
         assertMatches("C:/abs/path", true);
-        assertMatches("C:\\abs\\path", false);
         assertMatches("abs/path", false);
         assertMatches("C:/abs/path/", true);
     }
 
+    @Test
+    public void testListBasedMatcherMatchesAbsoluteWindowsStylePathWithForwardSlashWindows()
+    {
+        OsDetector.assumeRunningOnWindows();
+        createListMatcher("C:/abs/path");
```

Comment:
The created matcher is not visible in this code since it is a instance variable of the test. This makes the code hard to read and gives the programmer a chance to use the matcher before it is properly initialized. I would prefer a locally scoped variable here.

Reply:
Correct. But this will make the tests longer and harder to read. I added an explicit check that the matcher is initialized with a helpful error message.